<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BadgeWindow</class>
 <widget class="QMainWindow" name="BadgeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Badge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>BadgeWindow</receiver>
   <slot>selectedDay(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedDay(QDate)</slot>
 </slots>
</ui>
